skins/kdark/css/layout.css

```diff
@@ -1138,6 +1138,16 @@ box-shadow:  0px 1px 0px rgba(0, 0, 0, 0.3);
 	transform:         translate(-50%, -50%);
 	color: white;
 }
+
+.flashBlockAlertContainer{
```

Comment:
should not require a specific CSS should use the main CSS of the main error message. So that design updates to error message cleanly apply to all errors.